### PR TITLE
Make login modal

### PR DESCRIPTION
**Pull request by khvatskyi**

### CODE REVIEW

File: WorldAround.UI/src/app/components/authentication/login/login.component.html [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<form id="loginForm" class="form" (ngSubmit)="login()">
```
Comment (bigzerooo) [inline WorldAround.UI/src/app/components/authentication/login/login.component.html:1]: login-form

File: WorldAround.UI/src/app/components/authentication/authentication.scss [hunk 2]
```diff
@@ -0,0 +1,55 @@
+.form {
```
Comment (bigzerooo) [inline WorldAround.UI/src/app/components/authentication/authentication.scss:1]: Refactor for scss https://sass-lang.com/guide

File: WorldAround.UI/src/app/components/authentication/login/login.component.html (hunk 1 above)
Comment (bigzerooo) [inline WorldAround.UI/src/app/components/authentication/login/login.component.html:1]: login-form

File: WorldAround.UI/src/app/components/authentication/login/login.component.html [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+  <div>
+    <input #logUserName name="login" class="form-control" type="text" placeholder="username"
+      [(ngModel)]="loginModel.login" />
+    <input id="logPass" name="password" class="password" type="password" id="password" placeholder="password"
```
Comment (bigzerooo) [inline WorldAround.UI/src/app/components/authentication/login/login.component.html:5]: log-pass

File: WorldAround.UI/src/app/components/authentication/login/login.component.html [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+      [(ngModel)]="loginModel.password" />
+    <br>
+    <br>
+    <button type="submit" [disabled]="loginBtnDisabled">LOGIN</button>
+  </div>
+
+  <button id="toSignUp" (click)="openSignUp()" type="button">SIGN UP</button>
```
Comment (bigzerooo) [inline WorldAround.UI/src/app/components/authentication/login/login.component.html:12]: same

File: WorldAround.UI/src/app/components/authentication/login/login.component.ts [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+  styleUrls: ['./login.component.scss', '../authentication.scss']
+})
+export class LoginComponent implements OnDestroy, AfterViewInit {
+
+  @ViewChild('logUserName') userName: ElementRef;
+
+  public loginModel: LoginModel = new LoginModel();
+  public loginBtnDisabled: boolean = false;
+
+  constructor(private router: Router,
```
Comment (bigzerooo) [inline WorldAround.UI/src/app/components/authentication/login/login.component.ts:21]: private readonly

File: WorldAround.UI/src/app/components/authentication/signup/signup.component.html [hunk 6]
```diff
@@ -0,0 +1,18 @@
+<form id="signUpForm" class="form" (ngSubmit)="onSubmit()">
```
Comment (bigzerooo) [inline WorldAround.UI/src/app/components/authentication/signup/signup.component.html:1]: same

File: WorldAround.UI/src/app/components/authentication/login/login.component.ts [hunk 7]
```diff
@@ -0,0 +1,60 @@
...
+
+  ngAfterViewInit() {
+    this.userName.nativeElement.focus();
+  }
+
+  ngOnDestroy() {
+    this.toastr.toastrConfig.positionClass = 'toast-top-right';
+  }
+
+  openSignUp() {
```
Comment (bigzerooo) [inline WorldAround.UI/src/app/components/authentication/login/login.component.ts:37]: : void

File: WorldAround.UI/src/app/components/authentication/login/login.component.ts [hunk 8]
```diff
@@ -0,0 +1,60 @@
...
+  @ViewChild('logUserName') userName: ElementRef;
+
+  public loginModel: LoginModel = new LoginModel();
+  public loginBtnDisabled: boolean = false;
+
+  constructor(private router: Router,
+    private authService: AuthorizationService,
+    private toastr: ToastrService,
+    private dialogRef: MatDialogRef<LoginComponent>,
+    private dialog: MatDialog,) {
```
Comment (bigzerooo) [inline WorldAround.UI/src/app/components/authentication/login/login.component.ts:25]: ,

File: WorldAround.UI/src/app/components/authentication/login/login.component.ts [hunk 9]
```diff
@@ -0,0 +1,60 @@
...
+@Component({
+  selector: 'app-login',
+  templateUrl: './login.component.html',
+  styleUrls: ['./login.component.scss', '../authentication.scss']
+})
+export class LoginComponent implements OnDestroy, AfterViewInit {
+
+  @ViewChild('logUserName') userName: ElementRef;
+
+  public loginModel: LoginModel = new LoginModel();
```
Comment (bigzerooo) [inline WorldAround.UI/src/app/components/authentication/login/login.component.ts:18]: is public by default

File: WorldAround.UI/src/app/components/authentication/signup/signup.component.ts [hunk 10]
```diff
@@ -0,0 +1,63 @@
...
+@Component({
+  selector: 'app-signup',
+  templateUrl: './signup.component.html',
+  styleUrls: ['./signup.component.scss', '../authentication.scss']
+})
+export class SignupComponent implements OnDestroy, AfterViewInit {
+
+  @ViewChild('email') email: ElementRef;
+
+  public model: RegistrationModel = new RegistrationModel();
```
Comment (bigzerooo) [inline WorldAround.UI/src/app/components/authentication/signup/signup.component.ts:18]: Is public by default

**Review by bigzerooo — APPROVED** [review]

File: WorldAround.UI/src/app/components/authentication/login/login.component.ts (hunk 7 above)
Comment (khvatskyi) [reply WorldAround.UI/src/app/components/authentication/login/login.component.ts]: done

File: WorldAround.UI/src/app/components/authentication/login/login.component.ts (hunk 9 above)
Comment (khvatskyi) [reply WorldAround.UI/src/app/components/authentication/login/login.component.ts:18]: I suppose it will be much clear to specify the public

**Pull request merged**